app/modules/gdbstub.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Module for doing various horrible things 
```

Comment:
My bad -- I started off with a small private module that I wrote and I forgot to change the comment!